source/sas_compress.cpp

```diff
@@ -83,9 +83,6 @@ class LZ4Compressor : public SAS::Compressor
   static SAS::Compressor* get();
 
 private:
-  // The maximum size of input we can process in one go.
-  static const int MAX_INPUT_SIZE = 4096;
```

Comment:
(The comment, the name, the 'static' keyword and the 'const' keyword are all inappropriate now.)